e2e/pages/events/addHearingBookingDetailsEvent.page.js

```diff
@@ -0,0 +1,58 @@
+const I = actor();
+
+module.exports = {
+  fields: function (index) {
+    return {
+      hearingBooking: {
+        hearingType: {
+          caseManagement: `#hearingDetails_${index}_hearingType-CASE_MANAGEMENT`,
+        },
+        hearingVenue: `#hearingDetails_${index}_hearingVenue`,
+        date: {
+          day: `#hearingDetails_${index}_hearingDate-day`,
+          month: `#hearingDetails_${index}_hearingDate-month`,
+          year: `#hearingDetails_${index}_hearingDate-year`,
+        },
+        preHearingAttendance: `#hearingDetails_${index}_preHearingAttendance`,
+        hearingTime: `#hearingDetails_${index}_hearingTime`,
+
+        hearingNeedsBooked: {
+          interpreter: `#hearingDetails_${index}_hearingNeededDetails-INTERPRETER`,
+          welsh: `#hearingDetails_${index}_hearingNeededDetails-SPOKEN_OR_WRITTEN_WELSH`,
+          somethingElse: `#hearingDetails_${index}_hearingNeededDetails-SOMETHING_ELSE`,
+        },
+        giveDetails: `#hearingDetails_${index}_hearingNeededGiveDetails`,
+        judgeTitle: `#hearingDetails_${index}_judgeTitle`,
+        judgeFullName: `#hearingDetails_${index}_judgeFullName`,
+      },
+    };
+  },
+
+  addHearingButton: '//*[@id="hearingDetails"]/div/button[1]',
+
+  addHearing() {
+    I.click(this.addHearingButton);
+  },
+
+  async enterHearingDetails(hearingDetails) {
+    const elementIndex = await this.getActiveElementIndex();
+
+    I.click(this.fields(elementIndex).hearingBooking.hearingType);
+    await I.fillField(this.fields(elementIndex).hearingBooking.hearingVenue, hearingDetails.hearingVenue);
```

Comment:
Strange this this field gives issues without await. No action needed.

Reply:
Yeah its something to do with the click I think, so the await allows for the whole word to be entered, without it tests fail ad hocly as the word is not entered in full.